dashboard/test/controllers/api_controller_test.rb

```diff
@@ -282,10 +284,7 @@ class ApiControllerTest < ActionController::TestCase
   test "should get lock state when no user_level" do
     script, level, lesson = create_script_with_lockable_lesson
 
-    get :lockable_state, params: {
-      section_id: @section.id,
-      script_id: script.id
-    }
+    get :lockable_state, params: {script_id: script.id}
```

Comment:
Since we saw some issues that needed a revert on lockable state, should we add a test that lockable state also uses cotaught sections?

Reply:
See my change at the top of this file. It changes `@section` so that the logged-in `@teacher` is now a co-teacher and _not_ the section owner. This applies to multiple tests.

Reply:
(Also the change to this line is just because the `get :lockable_state` API doesn't do anything with the `section_id` parameter so I removed it from both tests that call this API).